chain/network/src/routing.rs

```diff
@@ -262,6 +263,11 @@ pub struct RoutingTable {
     pub peer_forwarding: HashMap<PeerId, Vec<PeerId>>,
     /// Store last update for known edges.
     pub edges_info: HashMap<(PeerId, PeerId), Edge>,
+    /// Shared version of edges_info used by multiple threads
+    edges_info_shared: Arc<Mutex<HashMap<(PeerId, PeerId), u64>>>,
+    /// Queue of edges verified, but not added yes
+    pub edges_to_add_receiver: QueueReceiver<Edge>,
+    pub edges_to_add_sender: QueueSender<Edge>,
```

Comment:
Do you think it is better to put these three fields in a separate struct?

Reply:
We always use those fields with `self.routing_table.<FIELD>`, we don't even touch them inside `routing.rs`.
It makes sense to put them in a separate struct inside `PeerManager`.

Reply:
I put them to `EdgeVerifierHelper`. The purpose of those fields is to communicate with `EdgeVerifier`,  that's much better now. Thanks.